### DISCUSSION TITLE
Second Blade compilation pass hits JavaScript mention of @livewireScripts

### DISCUSSION THREAD

**Discussion by michiel-van-den-anker** [livewire/livewire#2815] [3. Feature ideas]
Hi, thank you for your amazing work.

I am in a situation where I do a second pass of Blade compilation, on HTML that contains the result of `@livewireScripts`.

This HTML contains the (helpful) console message warning about loading the assets more than once. But gets picked up by another pass of Blade.

The line in question:
https://github.com/livewire/livewire/blob/69575f50bb7f8a49a41f9bd6bd16c73a6ef4fda3/src/LivewireManager.php#L251

I'm filtering this from the intermediate HTML now, but the @ symbol could be replaced with \u0040 to prevent this issue.

**Comment by calebporzio** [comment]
Ah, that's interesting. I'd merge a PR for that fix
